Docs/source/developers/collisions.rst

```diff
@@ -0,0 +1,80 @@
+.. _developers-collisions:
```

Comment:
I'm happy to move it to the theory chapter. I intended it to be a guide for how the collisions are implemented, specifically with regards to the equations used to calculate the collision energies and the transformations done to move between reference frames. Naturally having the equations in typeset would be much simpler for future users to get up to speed quickly rather than having to read through the code.
I do see that specifically with the inclusion of the difference in outcome between the classical and relativistic implementations, however that this would be better placed in the theory chapter.